### PR TITLE
Bug #211507 [WEB UI] : Prerak onboarding -> After uploading first photo - uploaded photo appearing on second photo upload page + The name of the file and type is going undefined after image compression for Prerak profile photo upload.

### PR DESCRIPTION
**Pull request by chaitanyakole**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [ ] Code is formatted as per format decided
* [ ] Updated acceptance criteria in tracker
* [ ] Updated test cases in test-cases-tracker
https://tracker.tekdi.net/issues/211507

## Summary by CodeRabbit

- **Refactor**
	- Updated state management and import statements in the BeneficiaryProfilePhoto component.
	- Modified state management and navigation handling in the PhotoUpload component.

### CODE REVIEW

File: apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js [hunk 1]
```diff
@@ -86,20 +55,22 @@ export default function PhotoUpload({
...
-            <React.Fragment />
+            <Fragment />
           )}
           <FrontEndTypo.Primarybutton
             isLoading={loading}
             p="4"
             mt="4"
             onPress={() => {
               navigate(`/profile/edit/upload/${page + 1}`);
+              navigate(0);
```
[the remark this answers is not in the document]
Comment (chaitanyakole) [reply apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js:73]: not required

File: apps/front-end/src/pages/front-end/BenificiaryProfilePhoto.js [hunk 2]
```diff
@@ -26,12 +26,12 @@ export default function BenificiaryProfilePhoto() {
     }
   };
 
-  React.useEffect(async () => {
+  useEffect(async () => {
```
[the remark this answers is not in the document]
Comment (chaitanyakole) [reply apps/front-end/src/pages/front-end/BenificiaryProfilePhoto.js:29]: not need

File: apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js [hunk 3]
```diff
@@ -86,20 +55,22 @@ export default function PhotoUpload({
...
+            key={page}
             value={formData?.[`profile_photo_${page}`]?.id}
             onChange={(e) => console.log(e)}
           />
           {errors?.fileSize ? (
             <H2 color="red.400">{errors?.fileSize}</H2>
           ) : (
-            <React.Fragment />
+            <Fragment />
           )}
```
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js:66]: remove Fragment

File: apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js (hunk 1 above)
Comment (sagarkoshti1990) [inline apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js:73]: remove navigate(0)

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js (hunk 3 above)
Comment (chaitanyakole) [reply apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js]: resolved

File: apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js (hunk 1 above)
Comment (chaitanyakole) [reply apps/front-end/src/pages/front-end/facilitator/edit/PhotoUpload.js]: resolve

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**